docker/Dockerfile

```diff
@@ -35,17 +35,22 @@ RUN apt-get install -y --no-install-recommends \
         unzip \
         zstd \
         zip \
-        zsh
-RUN pip3 install torch==1.12.1+cu116 -f https://download.pytorch.org/whl/torch_stable.html && \
+        zsh \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Small amount of space saved by merging `apt-get update && apt-get clean && rm -rf /var/lib/apt/lists/*` into the same `RUN` step.